### ISSUE TITLE
Missing song title in player, cover

### ISSUE THREAD

**Issue by lnicola** [agersant/polaris-web#43]
For me (Firefox/Linux), the player looks like:

![image](https://user-images.githubusercontent.com/308347/73604686-5830da00-459d-11ea-8ae4-56963b7a6ea2.png)

The file has metadata and an embedded cover (I don't know if Polaris can extract those):

```
    album           : Moderato cantabile
    artist          : Anja Lechner
    genre           : Classical
    title           : Voyage
    track           : 2
    date            : 2014

    Stream #0:1: Video: mjpeg (Baseline), yuvj420p(pc, bt470bg/unknown/unknown), 500x500 [SAR 1:1 DAR 1:1], 90k tbr, 90k tbn, 90k tbc (attached pic)
    Metadata:
      comment         : Cover (front)
```

![image](https://user-images.githubusercontent.com/308347/73604707-c9708d00-459d-11ea-938c-7f7771ce9688.png)

I also see an HTTP request to `/api/serve/null`, probably for the artwork.

**Comment by agersant** [comment]
Good catch. I'll keep this ticket open to avoid requesting `null`. The ticket for embedded art is https://github.com/agersant/polaris/issues/34.

**Comment by lnicola** [comment]
Did you notice that the title also shows up as `null`? I'm not sure why that happens.

**Comment by agersant** [comment]
That must be the last minute changes I did to clean up text formatting - should be super easy to fix. 

Thanks for testing the new version so quickly.

**Comment by agersant** [comment]
Titles fixed by ea4aee74af78b7555487b72ab101ea99346639c3

**Comment by lnicola** [comment]
The title shows up fine now, but the image still has `/api/serve/null` on the initial load. Clicking on songs doesn't do extra requests, though.

**Comment by agersant** [comment]
Fixed null art requests here: 2e11f282b73ad501546010ba11d842659b4e78ac
